storage/reads/types.tmpldata

```diff
@@ -2,26 +2,97 @@
 	{
 		"Name":"Float",
 		"name":"float",
-		"Type":"float64"
+		"Type":"float64",
```

Comment:
This is interesting only in the fact that this had to be added now. What changed that meant we needed to implement this now instead of long ago?

Reply:
There are a bunch of `.tmpldata` files scattered in the OSS repo and a lot of them have duplicated content.
My intention is to eliminate those duplicates, but since this PR has been open for a long time and there are several conflicts that I do not want to resolve here. So this part of the change is reverted.